gradle/libs.versions.toml

```diff
@@ -9,8 +9,8 @@ jaeger-core = '1.8.1'
 opentracing-grpc = '0.2.3'
 opentracing-mock = '0.33.0'
 
-micronaut = "4.0.0"
-micronaut-platform = "4.0.0-RC1"
+micronaut = "4.1.10"
+micronaut-platform = "4.1.5"
```

Comment:
🤔 this probably means we should bump the minor of this module (and create a 4.0.x branch)